components/generic/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,23 @@
+type CheckBoxProps = {
+  name: string;
+  label: string;
+  description?: string;
+};
+
+export const Checkbox = ({ name, label, description }: CheckBoxProps) => {
+  return (
+    <div className="flex flex-col gap-1.5">
```

Comment:
I would prefer to have consistent format with other input, which is `<label>` wraps `<input>` and all the necessary styling is added inside the `<label>`.